boards/waspmote-pro/Makefile.include

```diff
@@ -10,14 +10,8 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbserial*)))
 BAUD        ?= 9600
 include $(RIOTMAKE)/tools/serial.inc.mk
 
-# PROGRAMMER defaults to stk500v1 which is the internal flasher via USB
-# using avrdude. Can be overridden for debugging (which requires changes
-# that require to use an ISP)
-PROGRAMMER ?= stk500v1
-# configure programmer speed in baud
-FFLAGS_EXTRA += -b 115200
-# avoid error if mcu signature doesn't match
-FFLAGS_EXTRA += -F
```

Comment:
You dropped something there.
Not sure if those are needed though.

Reply:
115200 is the default speed. But I could add this anyway; maybe a but more verbosity is good as this would also document the differences if the bootloaders.

-F forces avrdude to flash even if the MCU is detected to be of different type. That is a bad idea to use that flag by default. If a user really wants to do this, she/he should do so manually and knowingly accept the risks.

Reply:
Yea I imagined you dropped some redundancy there 
Better to keep the Makefiles clean from unnecessary noise

Reply:
Sorry, read that to late. But I know undid adding the 115200 baud again.